packages/@uppy/robodog/src/addDashboardPlugin.js

```diff
@@ -36,7 +36,7 @@ function addDashboardPlugin (uppy, opts, overrideOpts) {
   })
 
   const inline = overrideOpts.inline == null ? dashboardOpts.inline : overrideOpts.inline
-  if (inline === false) {
+  if (!inline) {
```

Comment:
@arturi this is where we were checking it inside Robodog to determine whether modal options should be passed through